### PR TITLE
Maksym.Demianovskyi

### PR DESCRIPTION
**Pull request by m1ks-wq**

### CODE REVIEW

File: 02-Git/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+cmake_minimum_required(VERSION 3.16)
+project(02-Git)
+
+set(CMAKE_CXX_STANDARD 14)
+
+add_executable(02-Git main.cpp)
```
Comment (an1kh) [inline 02-Git/CMakeLists.txt]: no empty line at the end of the file

File: 02-Git/main.cpp [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#include <iostream>
+
+using namespace std;
+
+int main() {
+
+    cout << endl << "Rock, Paper, Scissors" << endl;
+
+}
```
Comment (an1kh) [inline 02-Git/main.cpp]: no empty line at the end of the file

**Review by an1kh — CHANGES_REQUESTED** [review]
Also please reword commit titles according to presentation
e.g. 02-Git: Add instead of added and so on.

**Comment by m1ks-wq** [comment]
> Also please reword commit titles according to presentation
> e.g. 02-Git: Add instead of added and so on.

reworded

File: 02-Git/CMakeLists.txt (hunk 1 above)
Comment (m1ks-wq) [reply 02-Git/CMakeLists.txt]: added

File: 02-Git/main.cpp (hunk 2 above)
Comment (m1ks-wq) [reply 02-Git/main.cpp]: added

File: 02-Git/main.cpp [hunk 3]
```diff
@@ -87,4 +89,15 @@ int main() {
             win();
         }
     }
-}
```
Comment (an1kh) [inline 02-Git/main.cpp]: Empty line is removed in last commit. please remove it first commit.
So this particular change will disappear.

**Review by an1kh — CHANGES_REQUESTED** [review]
Also commit " 02-Git: implemented rock-paper-scissors console game" still has incorrect title.

**Review by an1kh — APPROVED** [review]
Well done! Please merge

**Pull request merged**